ITEM_en.tsv

```diff
@@ -100,9 +100,9 @@ ITEM_20150317_000110	Unpopular during the Great Plant Cataclysm, the resurgence
 ITEM_20150317_000111	Chapparition Cutter
 ITEM_20150317_000112	According to a theory, this sword is also used in battles amongst the Demons.
 ITEM_20150317_000113	Deathweaver Cutter
-ITEM_20150317_000114	Thou, wielder of this blade, shalt become the deathbringer. / Now that you wield this blade, you will become the deathbringer.
+ITEM_20150317_000114	Thou, wielder of this blade, shalt become the deathbringer. Now that you wield this blade, you will become the deathbringer.
 ITEM_20150317_000115	Chapparition Sword
-ITEM_20150317_000116	This is the indirect evidence that there were disputes between Demons.
+ITEM_20150317_000116	This serves as indirect evidence for the disputes between Demons.
```

Comment:
Grammatically incorrect. The evidence talked about here shows that there were disputes between Demons, it (the evidence) was not used FOR the disputes between demons. In any case the grammar is incorrect/ambiguous. It should either be as is or "indirect evidence of the". As is is more clear as to what exactly the statement is saying.